newsroom/auth/views.py

```diff
@@ -367,7 +367,7 @@ def set_locale():
 
 @blueprint.route("/auth/impersonate", methods=["POST"])
 @admin_only
-def impersonate_user():
+async def impersonate_user():
```

Comment:
If a function does not use any async code, then it's not required to turn that function into async itself